### PR TITLE
W-12443157-APISharingUpdates-ESB

### PR DESCRIPTION
**Pull request by emileeburkeVT**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc [hunk 1]
```diff
@@ -315,7 +315,7 @@ The *Anypoint Platform* page appears.
 . In the navigation pane, click xref:access-management::managing-your-account.adoc[*Access Management*]. +
 The *Access Management* page appears.
 
-. In the *Access Management* page, in the navigation pane, click *Composer Sync*. +
+. In the *Access Management* page, in the navigation pane, click *Composer Sync*. If the *Composer Sync* button is not available, first click *Try New Features* to enable the button.+
```
Comment (hannanelson) [inline en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:318]:
```suggestion
. In the *Access Management* page, in the navigation pane, click *Composer Sync*. If the *Composer Sync* button is not available, first click *Try New Features* to enable the button. +
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc [hunk 2]
```diff
@@ -330,7 +330,7 @@ The *Account* page appears, displaying a linking request from Anypoint Platform.
 . In the *Access Token* section, click *Authenticate in Anypoint*. +
 The Anypoint Platform login page appears.
 
-. Log in to Anypoint Platform.
+. Log in to Anypoint Platform. Ensure that the user logging into Anypoint Platform has the right https://docs.mulesoft.com/api-manager/2.x/access-client-app-id-task[permissions to create a client app].
```
Comment (hannanelson) [inline en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:333]: I believe you can specify the version thusly:
`xref:2.x@api-manager::access-client-app-id-task[]`

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc [hunk 3]
```diff
@@ -387,6 +389,8 @@ The *Access Management* page appears.
 . In the navigation pane, click *Connected Apps*. +
 The *Connected Apps* page appears.
 
+. Optionally, on the *Connected Apps* page, if you would like to remove Composer’s access to a single API, click the options button next to the API and then click *Remove*.
```
Comment (hannanelson) [inline en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:392]:
```suggestion
. Optionally, on the *Connected Apps* page, if you want to remove Composer’s access to a single API, click the options button for the API and then click *Remove*.
```

File: en/salesforce/src/docs/ms_composer_release_notes_2023.adoc [hunk 4]
```diff
@@ -4,9 +4,16 @@ Use MuleSoft Composer for Salesforce to build connected experiences faster. You
...
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
-
 See xref:ms_composer_release_notes_ki.adoc[Known Issues for MuleSoft Composer for Salesforce] for a list of known issues by date.
 
+== January 31, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* The *Composer Administrator* permission set is now available. During the installation of Composer, the user completing the installation will automatically be assigned this permission. With this permission set, you can adjust the permissions of other users within your organization and view the *Settings* page.
```
Comment (hannanelson) [inline en/salesforce/src/docs/ms_composer_release_notes_2023.adoc:13]:
```suggestion
* The *Composer Administrator* permission set is now available. During the installation of Composer, the user completing the installation is automatically assigned this permission. With this permission set, you can adjust the permissions of other users within your organization and view the *Settings* page.
```

**Review by hannanelson — APPROVED** [review]

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc (hunk 2 above)
Comment (emileeburkeVT) [reply en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:333]: Thank you!

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc [hunk 5]
```diff
@@ -66,7 +66,9 @@ Salesforce certificates expire after one year. To renew an expired certificate,
 +
 MuleSoft creates and configures an instance of Anypoint Platform to provide a dedicated environment for Composer.
 . In the *Assign Users to MuleSoft Composer* section, click the *Assign Users* button to go directly to the Salesforce *Permission Sets* page:
-   .. Click *Composer User*.
+   .. Click *Composer User* or *Composer Administrator*.
+   ... *Composer User* has general acces to Composer funcionality such as building and managing flows.
```
Comment (bjoson-mulesoft) [inline en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:70]: access

File: en/salesforce/src/docs/ms_composer_release_notes_2023.adoc [hunk 6]
```diff
@@ -4,9 +4,19 @@ Use MuleSoft Composer for Salesforce to build connected experiences faster. You
...
+== February 21, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* The *Composer Administrator* permission set is now available. After the installation of Composer, the system administrator can assign the following permissions to users: 
+** The *Composer Administrator* has access to the *Settings* page and can manage email preferences and API Sharing.
+** The *Composer User* has general access to Composer functionality such as building and managing flows.
+
+
+* The *Settings* page is now available for those with the *Composer Administrator* permission set.  On this page, you can view and edit email notification settings.
```
Comment (bjoson-mulesoft) [inline en/salesforce/src/docs/ms_composer_release_notes_2023.adoc:18]: set is not really needed after permission

**Review by bjoson-mulesoft — APPROVED** [review]

File: en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc (hunk 5 above)
Comment (emileeburkeVT) [inline en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc:70]:
```suggestion
   ... *Composer User* has general access to Composer funcionality such as building and managing flows. 
```

File: en/salesforce/src/docs/ms_composer_release_notes_2023.adoc (hunk 6 above)
Comment (emileeburkeVT) [inline en/salesforce/src/docs/ms_composer_release_notes_2023.adoc:18]:
```suggestion
* The *Settings* page is now available for those with the *Composer Administrator* permission.  On this page, you can view and edit email notification settings.
```

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**